### DISCUSSION TITLE
Loop Cardinality for Multi-Instance

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#229] [Q&A] [answered]
Hi @ralphhanna,

When declaring multi-instance, how to specify to iterate `n` times.

Something like this: `$[...Array(n+1).keys()].slice(1)` works but is there any simpler way?

Also, how to use `loop`?
![image](https://github.com/user-attachments/assets/eb6705c3-3360-49c0-9a00-d4f81007e8a8)

**Comment by shashank-yelluri** [comment, marked as answer]
Hello @MaheshkumarSundaram 
You can refer the bpmn server documentation to understand more on the multi-instance configurations. Also, please try to check the `bpmn-web` code. There are some example provided by which you can understand the concept more easily. 

https://bpmnserver.github.io/docs/examples/loop/
